consensus/tendermint/tendermint.go

```diff
@@ -116,6 +117,8 @@ type Broadcasters[V Hashable[H], H Hash, A Addr] struct {
 }
 
 type Driver[V Hashable[H], H Hash, A Addr] struct {
+	tmdb TMDB
```

Comment:
I'm wondering if we can do this instead:
```suggestion
	db DB
```

Reply:
Updated